manifest/example-backend.yaml

```diff
@@ -0,0 +1,92 @@
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: example-backend
+  labels:
+    app: example-backend
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: example-backend
+  strategy:
+    type: Recreate
+  template:
+    metadata:
+      labels:
+        app: example-backend
+    spec:
+      containers:
+        - name: example-backend
+          image: quay.io/kube-bind/example-backend:latest
+          command:
+            - /example-backend
+          args:
+            - --namespace-prefix=cluster
+            - --pretty-name=Example-Backend
+            - --consumer-scope=Namespaced
+            - --oidc-issuer-client-id=$(OIDC-ISSUER-CLIENT-ID)
+            - --oidc-issuer-client-secret=$(OIDC-ISSUER-CLIENT-SECRET)
+            - --oidc-issuer-url=$(OIDC-ISSUER-URL)
+            - --oidc-callback-url=$(OIDC-CALLBACK-URL)
+            - --listen-ip=127.0.0.1
+            - --listen-port=8080
+            - --tls-cert-file=/etc/kube-bind/tls/server/tls.crt
+            - --tls-key-file=/etc/kube-bind/tls/server/tls.key
+          env:
+            - name: OIDC-ISSUER-CLIENT-ID
+              valueFrom:
+                secretKeyRef:
+                  name: oidc-configs
+                  key: oidc-issuer-client-id
+            - name: OIDC-ISSUER-CLIENT-SECRET
+              valueFrom:
+                secretKeyRef:
+                  name: oidc-configs
+                  key: oidc-issuer-client-secret
+            - name: OIDC-ISSUER-URL
+              valueFrom:
+                secretKeyRef:
+                  name: oidc-configs
+                  key: oidc-issuer-url
+            - name: OIDC-CALLBACK-URL
+              valueFrom:
+                secretKeyRef:
+                  name: oidc-configs
+                  key: oidc-callback-url
+          resources:
+            limits:
+              cpu: '2'
+              memory: 2Gi
+            requests:
+              cpu: '1'
+              memory: 1Gi
+          volumeMounts:
+            - name: kube-bind-certs
+              mountPath: /etc/kube-bind/tls/server
+              readOnly: true
+            - name: kube-bind-kubeconfig
+              mountPath: /etc/kubernetes/kubeconfig
+              readOnly: true
+      volumes:
+        - name: kube-bind-certs
+          secret:
+            secretName: kube-bind-cert
```

Comment:
where is the cert coming from?

Reply:
I haven't updated the certs creation yet, was planning to add [cert-manager](https://cert-manager.io/docs/) for that. If they were empty, then it will run in non secure env